pyrsa/rdm/rdms.py

```diff
@@ -361,6 +362,19 @@ def load_rdm(filename, file_type=None):
     return rdms_from_dict(rdm_dict)
 
 
+def rank_transform(rdms):
+    """ applyes a rank_transform and generates a new RDMs object"""
+    dissimilarities = rdms.get_vectors()
+    dissimilarities = np.array([rankdata(dissimilarities[i])
+                                for i in range(rdms.n_rdm)])
+    rdms_new = RDMs(dissimilarities,
```

Comment:
This seems to be a common pattern. (getting out the vectors and then repacking it in a new RDMs object). One way to refactor this might be to pass a lambda function to a re-usable unpack-transform-repack function;

```python
def doSomethingToRdms(myLambda):
    dissimilarities = rdms.get_vectors()
    dissimilarities = myLambda(dissimilarities)
    return RDMs(dissimilarities, <etc etc>)
```
and 
```python
def rank_transform(rdms):
    return doSomethingToRdms(lambda d: np.array([rankdata(dissimilarities[i]) for i in range(rdms.n_rdm)])))
```